src/IsoSprite.js

```diff
@@ -141,6 +141,9 @@ Object.defineProperty(Phaser.Plugin.Isometric.IsoSprite.prototype, "isoX", {
     set: function (value) {
         this._isoPosition.x = value;
         this._depthChanged = this._isoPositionChanged = this._isoBoundsChanged = true;
+        if (this.body){
```

Comment:
I used this flag to indicate that the position cache (prev) was out of data. This allows modifying isoPosition in game.update.